### PR TITLE
plugins/rbenv: Added a new plugin for working with rbenv

### PR DESCRIPTION
**Pull request by leonovk**
Added a new plugin for working with rbenv. It registers the necessary paths, initializes the environment, and sets a useful alias.

### CODE REVIEW

File: plugins/rbenv/rbenv.plugin.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#! bash oh-my-bash.module
+
+if [ -d ~/.rbenv ]; then
```
Comment (akinomyoga) [inline plugins/rbenv/rbenv.plugin.sh:3]:
```suggestion
if [[ -d ~/.rbenv ]]; then
```

**Review by akinomyoga — APPROVED** [review]

**Comment by akinomyoga** [comment]
Thank you. I merged this PR.

By the way, I noticed that your previous PR #588, which added `plugins/cargo`, actually only contains aliases. Maybe it should have been `aliases/cargo`. What do you think?

**Comment by leonovk** [comment]
@akinomyoga Yes, you are right. A little later I will move cargo to aliases

**Comment by akinomyoga** [comment]
Thank you!

**Pull request merged**